dev-docs/bidders/brainx.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: bidder
+title: brainx
+description: Prebid brainx Bidder Adapter
+biddercode: brainx
+aliasCode: fileContainingPBJSAdapterCodeIfDifferentThenBidderCode
```

Comment:
please remove this aliasCode line -- it will break the Download page.

Reply:
OK, we have fixed this issue